common/src/main/java/com/wynntils/models/characterstats/CharacterStatsModel.java

```diff
@@ -89,6 +89,10 @@ public double getBlocksAboveGround() {
                         McUtils.player().blockPosition().getZ()))
                 .isAir()) {
             endY--;
+
+            if (endY < -64) {
```

Comment:
Could also be worth extracting the -64 to a constant and a comment about why this check needs to happen